NoteFind/Views/NoteCell.m

```diff
@@ -5,13 +5,19 @@
 //  Created by Khloe Wright on 7/8/22.
 //
 
+#import "AppDelegate.h"
+#import "OfflineDataManager.h"
 #import "NoteCell.h"
 
+
+@interface NoteCell ()
```

Comment:
Do we need this category since it doesn't do anything?

Reply:
done!